modules/nixos/grub.nix

```
@@ -0,0 +1,29 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.boot.loader.grub.catppuccin;
+
+  theme = with pkgs;
+    let
+      src = fetchFromGitHub {
+        owner = "catppuccin";
+        repo = "grub";
+        rev = "803c5df0e83aba61668777bb96d90ab8f6847106";
+        sha256 = "sha256-/bSolCta8GCZ4lP0u5NVqYQ9Y3ZooYCNdTwORNvR7M0=";
+      };
+    in runCommand "catppuccin-grub-theme" { } ''
+      mkdir -p $out
+      cp -r ${src}/src/catppuccin-${cfg.flavour}-grub-theme/* \
+        $out/
```

```suggestion
      mkdir -p "$out"
      cp -r ${src}/src/catppuccin-${cfg.flavour}-grub-theme/* "$out"/
```
`$out` isn't likely to contain any glob or IFS characters, but it can't hurt. That line also doesn't seem long enough to be worth splitting.